EMacco/abarian-backend
# #170468582 Fetch product by id

#### What does this PR do?
This PR allows a user to be able to fetch a specific product using the ID

#### Description of Task to be completed?
- check if product exist
- return product details

#### How should this be manually tested?
- checkout to this branch
- send a get request to `/products/:productId`
- details of the product should be returned

#### Any background context you want to provide?
- product should exist

#### What are the relevant pivotal tracker stories?
[#170468582](https://www.pivotaltracker.com/story/show/170468582)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
